subprojects/docs/src/docs/userguide/what_is_gradle.adoc

```diff
@@ -12,133 +12,112 @@
-
-[[five_things]]
-== Five things you need to know about Gradle
-
-Gradle is a flexible and powerful build tool that can easily feel intimidating when you first start. However, understanding the following core principles will make Gradle much more approachable and you will become adept with the tool before you know it.
+https://scans.gradle.com/[Build scans] provide extensive information about a build that you can use to identify issues.
+You can use build scans to identify problems with a build's performance, and even share them for debugging help.
 
-=== 1. Gradle is a general-purpose build tool
+== Concepts
 
-Gradle allows you to build any software, because it makes few assumptions about what you're trying to build or how it should be done. The most notable restriction is that dependency management currently only supports Maven- and Ivy-compatible repositories and the filesystem.
+It's helpful to know the following concepts before you dive into the details of Gradle.
 
-This doesn't mean you have to do a lot of work to create a build. Gradle makes it easy to build common types of project — say Java libraries — by adding a layer of conventions and prebuilt functionality through <<plugins#plugins,_plugins_>>. You can even create and publish custom plugins to encapsulate your own conventions and build functionality.
+=== Projects
 
-[[the_core_model_is_based_on_tasks]]
-=== 2. The core model is based on tasks
+Projects are the things that Gradle builds.
+Projects contain a build script, named `build.gradle` or `build.gradle.kts`.
+Builds scripts define tasks, dependencies, plugins, and other configuration for your build.
+A single project can contain many subprojects, which can each contain their own subprojects.
 
-Gradle models its builds as Directed Acyclic Graphs (DAGs) of tasks (units of work). What this means is that a build essentially configures a set of tasks and wires them together — based on their dependencies — to create that DAG. Once the task graph has been created, Gradle determines which tasks need to be run in which order and then proceeds to execute them.
+=== Tasks
 
-This diagram shows two example task graphs, one abstract and the other concrete, with the dependencies between the tasks represented as arrows:
-
-.Two examples of Gradle task graphs
-image::task-dag-examples.png[Example task graphs]
-
-Almost any build process can be modeled as a graph of tasks in this way, which is one of the reasons why Gradle is so flexible. And that task graph can be defined by both plugins and your own build scripts, with tasks linked together via the <<tutorial_using_tasks#sec:task_dependencies,task dependency mechanism>>.
+Tasks contain the logic that implements builds.
+In most use cases, you'll use or extend existing tasks, like the built-in Java `test` task.
+Gradle provides task implementations for many common build sytem goals, like running tests and creating archives.
```

Comment:
Maybe "tasks implementing" vs. task implementations, which makes me think this is speaking about OO types.

Reply:
```suggestion
Gradle provides tasks that implement many common build system goals, like running tests and creating archives.
```
How does this sound?